### ISSUE TITLE
Problems Spawning Bots

### ISSUE THREAD

**Issue by Predhead** [Melledy/RoR2-PlayerBots#15]
I have a long list of mods going at the moment, been slowly working through them to make sure they work. I will attach an image of these.

Issue is as simple as it sounds, I load into the game and bots do not spawn. I have in the configs they are to be treat as Players (though I don't see a "Spawn as Players" option). 
The startup selection includes Captain, Commando (orig. game) a Paladin (one mod I have installed) and a Croc (a mod I don't have). Neither of these spawn.
Also console commands are being recognised, i.e. if I use addrandombot I get response " ... spawned 1 bot". However, the bot doesn't appear (tried using teleport).

Wondering if I am missing something or if there is an incompatibility that I've accidentally glossed over.

Thanks for any time you spend helping ^^

- edit:
Removed all mods. I can now use commands to spawn them, but still will not spawn on startup

- edit, solved startup:
Having a play around with the stuff, I decided I would edit the config file rather than the setting in the ThunderStore launcher
It seems the issue is that the ThunderStore Launcher config settings do not change the startup bots. I was able to add in 2 engineers by going into the config file with Visual Studio and changing the (hidden) variable " Startingbots.Engibody ".
Basically, note that the ThunderStore Launcher is not changing your Startingbots variables

 - edit, solved? :
Seems that, having added in each mod very slowly afterwards, that it no longer has any issues other than the one described above. Strange. Closing after this remark.

![image](https://user-images.githubusercontent.com/22813388/115954612-d4d50e80-a4e9-11eb-9fe6-f6da017c74e5.png)
![image](https://user-images.githubusercontent.com/22813388/115954628-e6b6b180-a4e9-11eb-9fe1-60c3942c7dbc.png)
![image](https://user-images.githubusercontent.com/22813388/115954631-eb7b6580-a4e9-11eb-8262-ced47ecda49b.png)
